swiftui/PasswordlessSwiftUI/PasswordlessSwiftUI/ContentView.swift

```diff
@@ -88,10 +106,13 @@ struct CustomStyledButton: View {
   let action: () -> Void
 
   var body: some View {
-    Button(title, action: action)
-    .padding()
-    .frame(maxWidth: .infinity)
-    .accentColor(.white)
+    Button(action: action) {
+      Spacer()
+      Text(title)
+        .padding()
+        .accentColor(.white)
+      Spacer()
```

Comment:
Does this work as is? It's unclear how these are assembled - is this an HStack, VStack, ZStack? I'm assuming it's just a `Group` but it's not clear.